screen2.0/src/common/components/mainsearch/CelltypeAutocomplete.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState, useEffect } from "react"
+import Box from "@mui/material/Box"
+import TextField from "@mui/material/TextField"
+import Autocomplete from "@mui/material/Autocomplete"
+import Grid from "@mui/material/Grid"
+import Typography from "@mui/material/Typography"
+import { debounce } from "@mui/material/utils"
+import { useRouter } from "next/navigation"
+import { gql, useQuery } from "@apollo/client"
+
+export const CelltypeAutocomplete = (props) => {
+  const [value, setValue] = useState(null)
+  const [inputValue, setInputValue] = useState("")
+  const [options, setOptions] = useState([])
+  const [cellTypes, setCelltypes] = useState([])
+  const [loading, setLoading] = useState(false)
+  const router = useRouter()
+
+  useEffect(() => {
+    fetch("https://downloads.wenglab.org/databyct.json")
```

Comment:
I think this fetch is defined as getGlobals in lib/queries.ts. Would be good to use that code to minimize times a url is hard coded if data/URL changes.

Reply:
Also, the autocomplete does not show mouse data. Using getGlobals and passing assembly should work.